Oqtane.Client/Modules/Controls/Pager.razor

```diff
@@ -1,6 +1,6 @@
 @namespace Oqtane.Modules.Controls
-@inherits LocalizableComponent
-
+@inherits ModuleControlBase
+@inject IStringLocalizerFactory LocalizerFactory
```

Comment:
This is the first time ever I see someone inject the `IStringLocalizerFactory` instead of `IStringLocalizer`